From maintainer, John Horigan <john@glyphic.com>
Context Free has an optional ability to generate QuickTime
files using FFmpeg. This is disabled by default. This patch enables
FFmpeg in Context Free.

--- a/Makefile
+++ b/Makefile
@@ -84,14 +84,14 @@
 # Uncomment these lines to enable FFmpeg support
 #
 
-# COMMON_SRCS += ffCanvas.cpp
-# LIBS += avformat avcodec swscale swresample avutil z m x264 pthread dl
+COMMON_SRCS += ffCanvas.cpp
+LIBS += avformat avcodec swscale swresample avutil z m x264 pthread dl
 
 #
 # Comment out this line to enable FFmpeg support
 #
 
-COMMON_SRCS += ffCanvasDummy.cpp
+# COMMON_SRCS += ffCanvasDummy.cpp
 
 SRCS = $(DERIVED_SRCS) $(COMMON_SRCS) $(UNIX_SRCS)
 
